ocs_ci/ocs/constants.py

```diff
@@ -480,3 +481,7 @@
 # Cluster name limits
 CLUSTER_NAME_MIN_CHARACTERS = 5
 CLUSTER_NAME_MAX_CHARACTERS = 17
+
+# Root Disk size
+CURRENT_VM_ROOT_DISK_SIZE = '60'
```

Comment:
@vavuthu what is differences between current_vm_root_disk_size and vm_root_disk_size? Thanks for explanation. I see you have to rebase cause of some other PR got merged. Thanks

Reply:
`current_vm_root_disk_size` is what is being replaced by `vm_root_disk_size` in the `replace_content_in_file()` calls.

Reply:
> @vavuthu what is differences between current_vm_root_disk_size and vm_root_disk_size? Thanks for explanation. I see you have to rebase cause of some other PR got merged. Thanks
current_vm_root_disk_size is the disk size which come by default.

Reply:
> Does it mean that if in the file there is somewhere else mentioned 60 it will be unintentionally changed to the number 120? Shouldn't we change it to something more precise?
> 
> I am not sure how the variable in tf file exactly looks like, but lets say `vm_disk_size = 60` I think we should replacing whole string as is and not just the number 60 which is quite fragile from the reason mentioned above.
> 
> Can you give me example how the line we are replacing in tf file looks like?
> 
> Thanks

string look like ``` 'size             = 60'```
In the latest commit , replacing the whole string instead of value